### PR TITLE
Issue67 scoreboard

### LINKED ISSUE #67: Create view for playlist scoreboard

### PR DESCRIPTION
**Pull request by AnnaSkantz**
This PR adds:
- a scoreboard component that is used in gameOverView,  userProfile-page, and friendProfile-page. 
- ratings in userProfile-page, and friendProfile-page

resolves #67

### CODE REVIEW

File: music-quiz/src/presenters/scoreboardPopupPresenter.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    userID: "2",
+    username: "Astra2",
+    score: 9,
+  },
+  {
+    userID: "3",
+    username: "Anna",
+    score: 8,
+  },
+];
```
Comment (alevarn) [inline music-quiz/src/presenters/scoreboardPopupPresenter.js:21]: Does not look like this sample data is used?

File: music-quiz/src/presenters/userProfilePresenter.js [hunk 2]
```diff
@@ -41,6 +42,32 @@ export default function UserProfilePresenter(props) {
...
+        <ReactStars
+          count={5}
+          value={averageRating}
+          size={20}
+          activeColor="#ffd700"
+          edit={false}
+        />
+      </div>
+    );
+  }
```
Comment (alevarn) [inline music-quiz/src/presenters/userProfilePresenter.js:69]: If a playlist does not have any scores/ratings (the `/playerHistory` directory does not exist in firebase) then this code crashes.

File: music-quiz/src/views/scoreboardView.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import trophy from "../images/trophy.png";
+import { Image, ListGroup, Badge } from "react-bootstrap";
+import { Link } from "react-router-dom";
+
+export default function ScoreboardView(props) {
+  let sortedScores = Object.values(props.scores).sort(
+    (a, b) => a.score - b.score
```
Comment (alevarn) [inline music-quiz/src/views/scoreboardView.js:7]: For me it sorts in ascending order (lowest to highest), but what we want is highest to lowest right?

**Review by alevarn — APPROVED** [review]
LGTM! Fix my comments and then merge!

File: music-quiz/src/presenters/scoreboardPopupPresenter.js (hunk 1 above)
Comment (asta12) [reply music-quiz/src/presenters/scoreboardPopupPresenter.js:21]: That is correct, we forgot to remove it. We have removed it now.

File: music-quiz/src/presenters/userProfilePresenter.js (hunk 2 above)
Comment (asta12) [reply music-quiz/src/presenters/userProfilePresenter.js:69]: We fixed this by adding a text saying "No ratings" and "This playlist has not been played yet" so now it works when no one has played it yet.

File: music-quiz/src/views/scoreboardView.js (hunk 3 above)
Comment (asta12) [reply music-quiz/src/views/scoreboardView.js]: Yes, we flipped it. This is now fixed

**Pull request merged**